src/farmbot_arduino_controller.cpp

```diff
@@ -203,16 +203,16 @@ void setup()
     pinMode(Z_ENCDR_A_Q, INPUT_PULLUP);
     pinMode(Z_ENCDR_B_Q, INPUT_PULLUP);
 
-    //  pinMode(AUX_STEP_PIN, OUTPUT);
-    //  pinMode(AUX_DIR_PIN, OUTPUT);
-    //  pinMode(AUX_ENABLE_PIN, OUTPUT);
+    pinMode(AUX_STEP_PIN, OUTPUT);
+    pinMode(AUX_DIR_PIN, OUTPUT);
+    pinMode(AUX_ENABLE_PIN, OUTPUT);
 
     pinMode(LED_PIN, OUTPUT);
-    //  pinMode(VACUUM_PIN, OUTPUT);
-    //  pinMode(WATER_PIN, OUTPUT);
-    //  pinMode(LIGHTING_PIN, OUTPUT);
-    //  pinMode(PERIPHERAL_1_PIN, OUTPUT);
-    //  pinMode(PERIPHERAL_2_PIN, OUTPUT);
+    pinMode(VACUUM_PIN, OUTPUT);
+    pinMode(WATER_PIN, OUTPUT);
+    pinMode(LIGHTING_PIN, OUTPUT);
+    pinMode(PERIPHERAL_1_PIN, OUTPUT);
+    pinMode(PERIPHERAL_2_PIN, OUTPUT);
```

Comment:
@gabrielburnworth should these be renamed to 4 and 5 to match the silkscreening on Farmduino?